ruby-builder.js

```diff
@@ -44,7 +45,9 @@ async function downloadAndExtract(platform, engine, version, rubyPrefix) {
   const parentDir = path.dirname(rubyPrefix)
 
   await io.rmRF(rubyPrefix)
-  await io.mkdirP(parentDir)
+  if (!fs.existsSync(parentDir)) {
+    await io.mkdirP(parentDir)
+  }
```

Comment:
This will delay the error if e.g. `parentDir` is a file and not a directory.
Is the change needed? What does it gain?

Reply:
Not sure if it's caused by @actions/io, but Windows JRuby was trying to create 'D:\'.  Can't create a drive on Windows, and it exists also.

But, `const parentDir = path.dirname(rubyPrefix)`, so it can't be a file...

Also, the problem is caused by the update to @actions/io. Working on a PR.